samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart/Resources/adaptiveCardResponseJson.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "type": "AdaptiveCard",
+  "refresh": {
+    "action": {
+      "type": "Action.Execute",
+      "verb": "basicRefresh",
+      "title": "RefreshTitle"
+    }
+  },
+  "body": [
+    {
+      "type": "TextBlock",
+      "text": "Happy Testing",
+      "size": "Medium",
+      "weight": "Bolder"
+    },
+    {
+      "type": "TextBlock",
+      "text": "Refreshed Card",
+      "size": "Medium",
+      "weight": "Bolder"
+    },
+    {
+      "type": "TextBlock",
+      "text": "Welcome Universal Card",
+      "size": "Medium",
+      "weight": "Bolder"
+
+    },
+    {
+      "type": "TextBlock",
+      "text": "${authResult}",
+      "size": "Medium",
+      "weight": "Bolder"
+    }
+  ],
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "appId": "ecc9fe9e-03df-4597-9c7d-22828d713d68",
+  "version": "1.4"
```

Comment:
Check if this version is correct and supported

Also, why did we push app id ?

Add place holder and Update Readme file to

Reply:
Changes updated